<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDialog</class>
 <widget class="QDialog" name="AddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>Материнская карта</string>
   </property>
   <item>
    <property name="text">
     <string>Материнская карта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Видеокарта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Процессор</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Оперативная память</string>
    </property>
   </item>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
